src/views/Politicians.vue

```diff
@@ -53,6 +91,15 @@ export default {
           .toLowerCase()
           .includes(this.search.toLowerCase());
       });
+    },
+    // CABINET VIEW CODE
+    showCabinet() {
+      return process.env.NODE_ENV == "development";
+      // return this.mock_cabinet;
+    },
+    // CABINET VIEW CODE
+    getCabinet() {
```

Comment:
async getCabinet() {..}
..

async getCabinet() {
  this.cabinet = await getCabinet()
}

Reply:
this should be under methods